packages/manager/apps/pci-private-registry/src/pages/create/steps/RegionStep.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  StepComponent,
+  TilesInputComponent,
+} from '@ovh-ux/manager-react-components';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import {
+  ODS_THEME_COLOR_HUE,
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+} from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { useContext, useEffect, useMemo } from 'react';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useProjectLocalisation } from '@ovh-ux/manager-pci-common';
+import { PRIVATE_REGISTRY_CREATE_LOCATION_NEXT } from '@/pages/create/constants';
+import { StepEnum } from '@/pages/create/types';
+import { useGetCapabilities } from '@/api/hooks/useCapabilities';
+import { useStore } from '@/pages/create/store';
+
+export default function RegionStep() {
+  const { t: tCreate } = useTranslation('create');
+  const { t: tCommonField } = useTranslation('common_field');
+
+  const { tracking } = useContext(ShellContext)?.shell || {};
+
+  const { projectId } = useParams();
+  const navigate = useNavigate();
+
+  const store = useStore();
+
+  const { act } = store;
```

Comment:
i am wondering if "act" isn't a little bit too generic as a name and could be renamed to provide more context

from what i understand this object contains step manipulation function so maybe you can rename it to something more straightforward like "step" ?

step.open(StepEnum.REGION), step.unlock(...) ect

what do you thnk about it ?